packages/asc-ui/src/components/Select/Select.tsx

```diff
@@ -12,15 +12,20 @@ const Select = React.forwardRef<
       value,
       label,
       srOnly: srOnlyProp,
-      error: errorProp,
+      error,
       errorMessage,
       labelStyle,
-      errorStyle,
       ...otherProps
     },
     ref,
   ) => {
-    const error = errorProp || errorMessage
+    if (errorMessage) {
```

Comment:
I think we can either show this warning, and still keep the `FormLabelStyle` (and don't mark it in the changelog as breaking, but deprecated.) OR we can remove this and consider it a breaking change

Reply:
Let's remove it entirely.

Reply:
👍